docs/guides/headless-gatsby-tutorial.md

```diff
@@ -4,99 +4,127 @@ title: Creating a blog with Headless CMS + Gatsby
 sidebar_label: Creating a blog with Headless CMS + Gatsby
 ---
 
-In this tutorial you will learn how to use Gatsby with our Headless CMS.
+In this tutorial we will learn how to use [Gatsby](https://www.gatsbyjs.org/) with the [Webiny Headless CMS](/docs/webiny-apps/headless-cms/features/content-modeling).
 
-[Here is a sample gatsby blog using Webiny's Headless CMS](https://github.com/webiny/webiny-examples/blob/master/headlesscms-gatsby)   
+> All of the code shown in this tutorial is also hosted in our [GitHub](https://github.com/webiny/webiny-examples/blob/master/headlesscms-nextjs) repository.) repository.
 
 ## Prerequisites
 
-Make sure you have Webiny installed and your `admin` app is running with Headless CMS enabled. You will use it to create `Content Models` and `Publish` content.
+##### 1. Gatsby-cli 
 
-You also need to know your API's url. You receive it when running `yarn webiny deploy api` and it can also be found int your AWS Management Console.
+Install gatsby-cli using `npm install -g gatsby-cli`. We will use it to create our Gatsby Blog.
 
-Finally, install gatsby-cli using `npm install -g gatsby-cli`. We will use it to create our Gatsby Blog.
+##### 2. A Webiny Project
 
-## Creating a Blog Post model in Headless CMS
+First of all, make sure you have a working [Webiny project](/docs/get-started/quick-start) set up.
 
-Run the `admin` app and navigate to **Headless CMS > Content Models > Models**. I will use the default `production` environment.
+> When setting up a new project, currently, there are [two project templates](/docs/get-started/quick-start#3-template-specific-setup) you can choose from: `full` and `cms`. Both include the Headless CMS app by default.
```

Comment:
We removed this, so remove this chunk of text

Reply:
fixed

also fixed "First of all" which appeared in the 2nd prerequisite...